docs/ossm/ambient/README.md

```diff
@@ -0,0 +1,358 @@
+# Getting Started with Istio Ambient Mode (Tech Preview)
+
+This section provides a high-level overview and installation procedure for Istio's ambient mode on OpenShift Container Platform (OCP) using OpenShift Service Mesh 3.1.x.
+
+---
+
+## 1. Overview of Istio Ambient Mode
+
+Istio's ambient mode offers a sidecar-less approach to service mesh, simplifying operations and reducing resource consumption. Instead of injecting a sidecar proxy into every application pod, ambient mode utilizes a node-level **ZTunnel proxy** for secure, mTLS-enabled connections and an optional **Waypoint proxy** for advanced L7 functionalities.
+
+### 1.1 Istio Ambient Mode Architecture
+
+* **ZTunnel Proxy:** A per-node proxy that handles secure, transparent TCP connections (mTLS) for all workloads within its node. It operates at Layer 4, offloading mTLS and L4 policy enforcement from application pods.
+* **Waypoint Proxy:** An optional, dedicated proxy deployed per service account or namespace. It provides advanced L7 functionalities like traffic management, policy enforcement, and observability for workloads that require them. This allows for selective application of L7 features without the overhead of sidecars for all services.
+* **Istio-CNI:** The Istio CNI plugin is responsible for redirecting traffic to the `ztunnel` proxy on each node, enabling transparent interception without requiring modifications to application pods.
+
+### 1.2 Why Use Istio's Ambient Mode?
+
+Istio's ambient mode offers several benefits:
+
+* **Simplified Operations:** Eliminates the need for sidecar injection and management, reducing operational complexity and cognitive load.
+* **Reduced Resource Consumption:** By centralizing mTLS and L4 policy enforcement in the `ztunnel`, ambient mode significantly lowers resource overhead per pod.
+* **Incremental Adoption:** Allows for gradual adoption of service mesh features. Workloads can join the mesh at L4 for mTLS and basic policy, and then selectively opt-in for L7 features via `waypoint` proxies as needed.
+* **Enhanced Security:** Provides a secure, zero-trust network foundation with mTLS by default for all meshed workloads.
+
+**Trade-offs:**
+
+* Ambient mode is a newer architecture and may have different operational considerations compared to the traditional sidecar model.
+* L7 features require the deployment of `waypoint` proxies, which add a small amount of overhead for the services that utilize them.
+
+---
+
+## 2. Pre-requisites to Using Ambient Mode with OSSM 3
+
+Before installing Istio's ambient mode with OpenShift Service Mesh, ensure the following prerequisites are met:
+
+* **OpenShift Container Platform 4.15+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
```

Comment:
I notice below we say that 4.19 is required. 4.19 will be the first version of OCP with supported Gateway API CRDs. Should this say 4.19?

Reply:
Hi @longmuir 

I had the following comment from @sridhargaddam 

> In Line 37, we have specified the pre-requisite as OCP 4.19+ Whereas here we are suggesting 4.15, lets be consistent.
```
**OpenShift Container Platform 4.19+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
```